Source/ASCollectionNode.mm

```diff
@@ -464,6 +464,16 @@ - (void)setBatchFetchingDelegate:(id<ASBatchFetchingDelegate>)batchFetchingDeleg
   return _batchFetchingDelegate;
 }
 
+- (BOOL)usesSynchronousDataLoading
+{
+  return self.view.usesSynchronousDataLoading;
+}
+
+- (void)setUsesSynchronousDataLoading:(BOOL)usesSynchronousDataLoading
+{
+  self.view.usesSynchronousDataLoading = usesSynchronousDataLoading;
```

Comment:
In the future, this may actually trigger a premature view allocation. Let's handle it now by putting the flag in pending state if the node is not loaded yet.